src/Microsoft.ML.Data/Utilities/ColumnCursor.cs

```diff
@@ -99,7 +100,7 @@ private static IEnumerable<T> GetColumnDirect<T>(IDataView data, int col)
             Contracts.AssertValue(data);
             Contracts.Assert(0 <= col && col < data.Schema.Count);
 
-            using (var cursor = data.GetRowCursor(col.Equals))
+            using (var cursor = data.GetRowCursor(col))
```

Comment:
>col [](start = 50, length = 3)

Once converted to be over schema, something like this would be `data.Schema[col]`, if we are to do this.